Code/Framework/AzCore/AzCore/Memory/MallocSchema.cpp

```diff
@@ -144,6 +144,11 @@ AZ::MallocSchema::size_type AZ::MallocSchema::GetMaxAllocationSize() const
     return 0xFFFFFFFFull;
 }
 
+AZ::MallocSchema::size_type AZ::MallocSchema::GetMaxContiguousAllocationSize() const
+{
+    return 0xFFFFFFFFull;
```

Comment:
Nitpick: random magic number. Consider making a static constexpr value for this and explain why 4Gib is the maximum amount of memory (which seems arbitrary, especially on a 64-bit system).

Reply:
Well it isn't really a random magic number if it is being returned from a function as it's only statement.

I see the following as the same
- `constexpr AZ::MallocSchema::size_type MaxConfiguousAllocatorSize = 0xFFFFFFFFull`
- `constexpr AZ::MallocSchema::size_type GetMaxConfiguousAllocatorSize()`
They are both equivalent to constants.

Reply:
Well the return value is being changed to `AZ_CORE_MAX_ALLOCATOR_SIZE` anyway.

So "Soblem Prolved"